### ISSUE TITLE
OSM corona: Namensraum

### ISSUE THREAD

**Issue by Matthias84** [binary-butterfly/shared-delivery#34]
Andere Projekte wie https://github.com/osmontrouge/caresteouvert nutzen den "corona*:*" Namensraum, um temporäre Öffnungszeiten etc. direkt bei OSM einzupflegen. Besteht da evtl. die Möglichkeit der Zusammenarbeit, um einen möglichst große Nachnutzung der Nutzerbeiträge zu ermöglichen?

https://twitter.com/ClaasAug/status/1246137772907388929

**Comment by caugner** [comment]
Genau, bei https://www.caresteouvert.fr/ werden Tags mit dem Namensraum `covid19` verwendet:

* [opening_hours:covid19=*](https://taginfo.openstreetmap.org/keys/?key=opening_hours%3Acovid19)
* [description:covid19=*](https://taginfo.openstreetmap.org/keys/?key=description%3Acovid19)
* [delivery:covid19=*](https://taginfo.openstreetmap.org/keys/?key=delivery%3Acovid19)
* [takeaway:covid19=*](https://taginfo.openstreetmap.org/keys/?key=takeaway%3Acovid19)
* [note:covid19=*](https://taginfo.openstreetmap.org/keys/?key=note%3Acovid19)

**Comment by the-infinity** [comment]
Fragt sich, in wiefern das berücksichtigt werden muss bei uns? Aktuell gibt es kein automatisches Zurückspielen, und da die Kommunen auch erst einmal lernen müssen mit dem System umzugehen ist das vielleicht auch gar nicht so blöd.
